client/src/history.js

```diff
@@ -0,0 +1,2 @@
+import createHistory from 'history/createHashHistory';
+export default createHistory();
```

Comment:
Not sure if you need this file to redirect. Look into `withRouter` and `history.push`

Reply:
Nice...I've checked it and learned how to use it
Unfortunately, both method will be removed when sign up is removed